oneflow/python/ops/nn_ops.py

```diff
@@ -3245,3 +3245,33 @@ def _p_norm(x, p=2.0, name="p_norm"):
         return flow.math.reduce_sum(_triplet_loss, name=name + "_reduce_sum")
     else:
         return _triplet_loss
+
+
+@oneflow_export("nn.PixelShuffle")
+def pixel_shuffle(
+    input: remote_blob_util.BlobDef,
+    h_factor: int,
+    w_factor: int,
+    name: Optional[str] = None,
+) -> remote_blob_util.BlobDef:
+    assert (
+        h_factor > 0 and w_factor > 0
+    ), "The factor of height and width must larger than zero"
+    assert len(input.shape) == 4, "Only Accept 4D Blob"
+
+    _batch, _channel, _height, _width = input.shape
+    assert (
+        _channel % (h_factor * w_factor) == 0
+    ), "The channels of input must be divided by (h_factor * w_factor)"
```

Comment:
The channels of input tensor must be divisible by (h_upscale_factor * w_upscale_factor)